sci-electronics/drahnr-oregano/drahnr-oregano-0.84.40.ebuild

```diff
@@ -38,7 +44,9 @@ pkg_setup() {
 	fi
 }
```

Comment:
obliterate, `python-any-r1_pkg_setup` contains the `MERGE_TYPE` logic already

Reply:
Done